tests/test_utils/test_distances_utils.py

```diff
@@ -0,0 +1,10 @@
+import numpy as np
```

Comment:
A small comment. At the beginning of this module, in addition to the module docstring, we could add the licence. Just like in the other modules, see https://github.com/sissa-data-science/DADApy/blob/main/tests/test_id_estimation/test_2NN.py